core/arch/arm/kernel/link_dummies_paged.c

```diff
@@ -38,3 +40,18 @@ __wq_rpc(uint32_t func __unused, int id __unused,
 	 int lineno  __unused)
 {
 }
+
+const struct mobj_ops mobj_reg_shm_ops __rodata_dummy;
+const struct mobj_ops mobj_ffa_ops __rodata_dummy;
+const struct mobj_ops mobj_phys_ops __rodata_dummy;
+const struct mobj_ops mobj_virt_ops __rodata_dummy;
+const struct mobj_ops mobj_mm_ops __rodata_dummy;
+const struct mobj_ops mobj_shm_ops __rodata_dummy;
+const struct mobj_ops mobj_seccpy_shm_ops __rodata_dummy;
+const struct mobj_ops mobj_with_fobj_ops __rodata_dummy;
+const struct fobj_ops ops_rwp_paged_iv __rodata_dummy;
+const struct fobj_ops ops_rwp_unpaged_iv __rodata_dummy;
+const struct fobj_ops ops_ro_paged __rodata_dummy;
+const struct fobj_ops ops_ro_reloc_paged __rodata_dummy;
+const struct fobj_ops ops_locked_paged __rodata_dummy;
+const struct fobj_ops ops_sec_mem __rodata_dummy;
```

Comment:
Should be a separate commit

Reply:
I'm sorry I don't get it. What should be a separate commit?

Reply:
Sorry I reviewed the patches one by one but I submitted all comments at once so the context is unclear.
You are introducing this change in commit "core: make __rodata_unpaged() symbols __weak". This does not fit the description.

Reply:
The part I'm not getting is how it doesn't fit the description in "core: make __rodata_unpaged() symbols __weak".

Reply:
Ah yes, "in order to be overridden". Sorry, really not my day :cry: I think I'll stop reviewing stuff for now :roll_eyes: :laughing:

Reply:
No worries, we all have days like that. :-)